ocaml/libs/tracing/tracing_core.ml

```diff
@@ -11,7 +11,7 @@
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU Lesser General Public License for more details.
  *)
-module D = Debug.Make (struct let name = "tracing" end)
+module D = Debug.Make (struct let name = "tracing-core" end)
```

Comment:
you'll need to update /etc/xapi.conf.d/tracing.conf to include `tracing-core` there.

Reply:
I kept the original name for the core.